docs/Getting-Started/Private-Network-Quickstart.md

```diff
@@ -410,61 +431,66 @@ Compiling truffle/DeployedAddresses.sol...
   3 passing (37s)
 ```
 
-Continue with the regular tutorial steps in the [Creating a user interface to interact with the smart contract section](https://truffleframework.com/tutorials/pet-shop#creating-a-user-interface-to-interact-with-the-smart-contract).
+Continue with the Truffle tutorial steps in the [Creating a user interface to interact with the smart contract](https://truffleframework.com/tutorials/pet-shop#creating-a-user-interface-to-interact-with-the-smart-contract) section.
 
-We have already connected our private network to MetaMask so you can skip the [Installing and configuring MetaMask section](https://truffleframework.com/tutorials/pet-shop#installing-and-configuring-metamask).
+We've already connected the private network to MetaMask, so you can skip the [Installing and configuring MetaMask](https://truffleframework.com/tutorials/pet-shop#installing-and-configuring-metamask) section.
 
-Continue with the regular tutorial steps from the [Installing and configuring lite-server section](https://truffleframework.com/tutorials/pet-shop#installing-and-configuring-lite-server) to the end of the tutorial.
+Continue with the regular tutorial steps from the [Installing and configuring lite-server](https://truffleframework.com/tutorials/pet-shop#installing-and-configuring-lite-server)
+section and finish the tutorial.
 
-When you adopt pets in the browser and approve the transaction in MetaMask, you will be able to see the transactions in the block explorer. 
+When you adopt pets in the browser and approve the transaction in MetaMask, you'll be able to see the transactions in the block explorer.
 
-## Shut Down the Network and Remove the Containers
- 
-To shut down the network and delete all containers:
 
-```bash tab="Linux/macOS"
-# Shell script are provided in the Quickstart directory
+## Stop / Restart Private Network without Removing Containers 
 
-quickstart/removePantheonPrivateNetwork.sh
-```
+To shut down the private network without deleting the containers:
 
-```bat tab="Windows"
-// Run the docker-compose commands directly
+```bash tab="Linux/MacOS"
+# Stop the containers:
 
-quickstart\docker-compose down
+./stop.sh
 ```
 
-!!!note
-    On Windows, the quickstart creates a volume called `quickstart_public-keys` but it's not automatically removed. Remove this volume using `docker volume rm quickstart_public-keys`.
+```bash tab="Windows"
+// Stop the containers:
 
-## Stop and restart the Private Network without Removing the Containers 
+.\stop.sh
+```
+
+!!! note
```

Comment:
No, it will only stop the containers related to the services indicated in the docker-compose.yml file
By the way, we talk about services in docker-compose context and containers in docker context.